tomviz/MainWindow.h

```diff
@@ -81,6 +86,7 @@ private slots:
   void registerCustomOperators();
 
   QScopedPointer<Ui::MainWindow> m_ui;
+  QMenu* m_customTransformsMenu;
```

Comment:
I think we should prefer inline initialization to nullptr as below over doing so in the contructor

Reply:
Sounds good. Still not in the swing of C++11...

Reply:
Fixed in latest revision.